### PR TITLE
#172455545 Fix nodejs version issue

### PR DESCRIPTION
**Pull request by descholar-ceo**
### What does this PR ​ do​ ? 
Fix the nodejs v14 to nodejs v13 on travis

### Description of Task to be completed?
-Change the version of nodejs in .travis.yml file

### How should this be manually tested?
Check the TravisCI build, if it is passing all of the tests, then this PR is okay

### Any background context you want to provide?
Use [nodejs between v6 and v13](https://nodejs.org/en/) to run this project

### What are the relevant pivotal tracker stories?
[ ##172455545](https://www.pivotaltracker.com/n/projects/2435482/stories/#172455545)

### Screenshots 
N/A

### Questions:
N/A

### CODE REVIEW

**Review by Ugizwenayo-Divine — COMMENTED** [review]
hound is pending
re-push

**Comment by descholar-ceo** [comment]
> hound is pending
> re-push

@Ugizwenayo-Divine I pushed like two times but it keeps pending

File: .travis.yml [hunk 1]
```diff
@@ -20,4 +20,4 @@ notification:
   - email: false
 
 after_success:
-  - npm run coverage
\ No newline at end of file
+  - npm run coverage
```
Comment (kipropbrian) [inline .travis.yml:23]: What is changed here?

Comment (descholar-ceo) [reply .travis.yml:23]: I was a white space

Comment (since9629) [reply .travis.yml:23]: I would suggest not to change anything on this line

**Pull request merged**